### PR TITLE
Add test test_6541_4 for UGENE-6541

### PR DESCRIPTION
**Pull request by ichebyki**

### CODE REVIEW

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp [hunk 1]
```diff
@@ -2988,6 +2988,41 @@ GUI_TEST_CLASS_DEFINITION(test_6541_3) {
...
+    // Select 1 sequences in the alignment.
+    // Expected result : the "Realign sequence(s) to other sequences" button is enabled.
+    GTUtilsMsaEditor::selectRows(os, 18, 18);
+    CHECK_SET_ERR(realignButton->isEnabled(), "'Realign sequence(s) to other sequences' is unexpectably disabled");
+
+    // Click "Align sequence(s) to this alignment".
+    // Expected result : the "Realign sequence(s) to other sequences" button is disabled.
+    GTUtilsDialog::waitForDialog(os, new GTFileDialogUtils(os, testDir + "_common_data/alignment/align_sequence_to_an_alignment/TUB.msf"));
+    GTWidget::click(os, alignButton);
+    GTWidget::click(os, realignButton);
```
Comment (rasputinkirill) [inline src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp]: Remove 2 unnecessary clicks

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp [hunk 2]
```diff
@@ -2988,6 +2988,41 @@ GUI_TEST_CLASS_DEFINITION(test_6541_3) {
...
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+    CHECK_SET_ERR(undoButton->isEnabled(), "'Undo' button is unexpectably disabled");
+    GTWidget::click(os, undoButton);
+
+    // Click "Realign sequence(s) to other sequences".
+    // Expected result : the "Realign sequence(s) to other sequences" button is disabled.
+    GTUtilsMsaEditor::selectRows(os, 18, 18);
+    GTWidget::click(os, realignButton);
+    GTWidget::click(os, realignButton);
+    GTWidget::click(os, realignButton);
```
Comment (rasputinkirill) [inline src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp]: Remove 2 unnecessary clicks

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp [hunk 3]
```diff
@@ -2988,6 +2988,41 @@ GUI_TEST_CLASS_DEFINITION(test_6541_3) {
...
+    CHECK_SET_ERR(!realignButton->isEnabled(), "'Realign sequence(s) to other sequences' is unexpectably enabled");
```
Comment (rasputinkirill) [inline src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp:3016]: Change to "realignButton->isDisabled()" for better understanding.

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp [hunk 4]
```diff
@@ -2988,6 +2988,41 @@ GUI_TEST_CLASS_DEFINITION(test_6541_3) {
...
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+    CHECK_SET_ERR(undoButton->isEnabled(), "'Undo' button is unexpectably disabled");
```
Comment (rasputinkirill) [inline src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp]: Change to "realignButton->isDisabled()" for better understanding.

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp [hunk 5]
```diff
@@ -2988,6 +2988,41 @@ GUI_TEST_CLASS_DEFINITION(test_6541_3) {
...
+    GTWidget::click(os, undoButton);
```
Comment (rasputinkirill) [inline src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp]: Remove this. Useless code.

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp [hunk 6]
```diff
@@ -2988,6 +2988,41 @@ GUI_TEST_CLASS_DEFINITION(test_6541_3) {
...
+    GTUtilsMsaEditor::selectRows(os, 18, 18);
+    CHECK_SET_ERR(realignButton->isEnabled(), "'Realign sequence(s) to other sequences' is unexpectably disabled");
+
+    // Click "Align sequence(s) to this alignment".
+    // Expected result : the "Realign sequence(s) to other sequences" button is disabled.
+    GTUtilsDialog::waitForDialog(os, new GTFileDialogUtils(os, testDir + "_common_data/alignment/align_sequence_to_an_alignment/TUB.msf"));
+    GTWidget::click(os, alignButton);
+    GTWidget::click(os, realignButton);
+    GTWidget::click(os, realignButton);
+    CHECK_SET_ERR(!realignButton->isEnabled(), "'Realign sequence(s) to other sequences' is unexpectably enabled");
```
Comment (rasputinkirill) [inline src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp:3007]: Change to "realignButton->isDisabled()" for better understanding.

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp [hunk 7]
```diff
@@ -2988,6 +2988,41 @@ GUI_TEST_CLASS_DEFINITION(test_6541_3) {
     CHECK_SET_ERR(!realignButton->isEnabled(), "'Realign sequence(s) to other sequences' button is unexpectably enabled");
 }
 
+GUI_TEST_CLASS_DEFINITION(test_6541_4) {
+    //  1. Open "COI_SHORT_21x88.aln".
```
Comment (rasputinkirill) [inline src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp]: Incorrect comment. Fix it.

**Review by rasputinkirill — CHANGES_REQUESTED** [review]

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp (hunk 6 above)
Comment (ichebyki) [reply src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp:3007]: I don't know those function

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp (hunk 1 above)
Comment (ichebyki) [reply src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp]: done

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp (hunk 2 above)
Comment (ichebyki) [reply src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp]: done

**Review by rasputinkirill — APPROVED** [review]

**Pull request merged**